<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QWidget" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#menu{border-image: url(:/images/menu1.jpg);}

QPushButton {
    color: white;
    padding: 5px;
    border-radius: 15px;
	color: rgba(255, 255,255, 250);
	background-color: rgba(0, 33, 68,220);
	border:none; 
}

QPushButton:hover {
    background-color: #2F42B5;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #2F42B5;
}</string>
  </property>
  <widget class="QPushButton" name="swt_Control_bt">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>540</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>切换自动</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/auto_model.png</normaloff>:/images/auto_model.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="ctl_Frame_sw">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>53</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 33, 68,200);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="ctl_Panel_sw">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>53</y>
     <width>221</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 33, 68,120);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="turn_up_bt">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/up.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="autoRepeatDelay">
      <number>30</number>
     </property>
     <property name="autoRepeatInterval">
      <number>30</number>
     </property>
    </widget>
    <widget class="QPushButton" name="turn_reset_bt">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/reset.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="turn_left_bt">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/left.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="autoRepeatDelay">
      <number>30</number>
     </property>
     <property name="autoRepeatInterval">
      <number>30</number>
     </property>
    </widget>
    <widget class="QPushButton" name="turn_right_bt">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/right.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="autoRepeatDelay">
      <number>30</number>
     </property>
     <property name="autoRepeatInterval">
      <number>30</number>
     </property>
    </widget>
    <widget class="QPushButton" name="turn_down_bt">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/down.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="autoRepeatDelay">
      <number>30</number>
     </property>
     <property name="autoRepeatInterval">
      <number>30</number>
     </property>
    </widget>
    <widget class="QPushButton" name="get_photo_bt">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>420</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/camera.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="get_video_bt">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>420</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/capture.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="capture_bt">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    color: white;
    padding: 5px;
    border-radius: 15px;
	color: rgba(255, 255,255, 250);
	background-color: rgba(0, 33, 68,250);
	border:none; 
}

QPushButton:hover {
    background-color: #2F42B5;
}

QPushButton:pressed, QPushButton:checked {
    color: #2F42B5;
}</string>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mqtt_transfer_bt">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>330</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    color: white;
    padding: 5px;
    border-radius: 15px;
	color: rgba(255, 255,255, 250);
	background-color: rgba(0, 33, 68,250);
	border:none; 
}

QPushButton:hover {
    background-color: #2F42B5;
}

QPushButton:pressed, QPushButton:checked {
    color: #2F42B5;
}</string>
     </property>
     <property name="text">
      <string>图传</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QSlider" name="angle_slider">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>29</width>
       <height>271</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 33, 68,10);


</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="speed_slider">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>29</width>
       <height>271</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 33, 68,10);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="auto_btn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>自动旋转</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 33, 68,10);</string>
     </property>
     <property name="text">
      <string>角度控制</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 33, 68,10);</string>
     </property>
     <property name="text">
      <string>速度控制</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="exp_Check_bt">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>540</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>异常查看</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/exception.png</normaloff>:/images/exception.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="file_Manage_bt">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>540</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>文件管理</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/file_manager.png</normaloff>:/images/file_manager.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="mode_Setup_bt">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>540</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>模式设置</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="currentTime_lb">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fps_lbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(224, 18, 18);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
